server/app-engine/static/gtfs-rt-util.js

```diff
@@ -111,7 +122,7 @@ if (!fetch) {
     }
 
     // 's' is assumed to be a time string like '8:23 am'
-    function getTimeFromString(s) {
+    exports.getTimeFromString = function(s) {
```

Comment:
@unreasonableman do you know what the reason for this change was? I should have caught this before - I had written it this way deliberately since the function is only used within gtfs-rt-util.js and doesnt need be exported.